src/System.Memory/tests/Memory/Trim.cs

```diff
@@ -90,7 +90,9 @@ public static void MemoryExtensions_Trim_Single(int[] values, int trim, int[] ex
         [InlineData(new int[0], new int[0], new int[0])]
         [InlineData(new int[0], new int[] { 1 }, new int[0])]
         [InlineData(new int[] { 1 }, new int[0], new int[] { 1 })]
+#pragma warning disable xUnit1025 // InlineData duplicates
```

Comment:
Looks like an analyser bug unless I can't see properly.

Reply:
The rest of the same suppressions are for the same reason.

Reply:
💡 I didn't see a duplicate of this; consider filing an issue

Reply:
Existing issue, left a comment there: https://github.com/xunit/xunit/issues/1877